### DISCUSSION TITLE
Gaming/Regionless Platform Use Case Discussion

### DISCUSSION THREAD

**Discussion by zmn223** [CentaurusInfra/fornax#94] [General]
Centaurus Community Meeting Discussion 
[Join Meeting](https://docs.google.com/document/d/1Cwpp44pQhMZ_MQ4ebralDHCt0AZHqhSkj14kNAzA7lY/edit)

Action Items:
![MQ](https://user-images.githubusercontent.com/50963591/159385479-25406ad8-85b2-496c-bf1e-16dcc28e3496.PNG)

**Comment by ysun3-futurewei** [comment]
Design proposal
[Online Game Stream Processing Platform.pptx](https://github.com/CentaurusInfra/fornax/files/8375382/Online.Game.Stream.Processing.Platform.pptx)

**Comment by pdgetrf** [comment]
[regionless storage scoping](https://github.com/CentaurusInfra/fornax/blob/main/docs/fornax-design/open%20source%20meeting%200329.pptx)
